components/rpill/buildflags.gni

```diff
@@ -0,0 +1,5 @@
+import("//build/config/features.gni")
+
+declare_args() {
+  rpill = is_mac || is_win
```

Comment:
I don't see anything enabled for mac, the rest of the code has `is_win` guards. Also this should be `enable_rpill`

Reply:
Changed to `enable_rpill`